clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/AwsInfrastructureProvider.groovy

```diff
@@ -23,11 +23,12 @@ import com.netflix.spinnaker.clouddriver.aws.AmazonCloudProvider
 import com.netflix.spinnaker.clouddriver.cache.KeyParser
 import com.netflix.spinnaker.clouddriver.cache.SearchableProvider
 import com.netflix.spinnaker.clouddriver.aws.cache.Keys
+import com.netflix.spinnaker.clouddriver.security.BaseProvider
 
 import static com.netflix.spinnaker.clouddriver.aws.cache.Keys.Namespace.SECURITY_GROUPS
 import static com.netflix.spinnaker.clouddriver.cache.SearchableProvider.SearchableResource
 
-class AwsInfrastructureProvider extends AgentSchedulerAware implements SearchableProvider {
+class AwsInfrastructureProvider extends AgentSchedulerAware, BaseProvider implements SearchableProvider {
```

Comment:
- Remove `AgentSchedulerAware` (it's in `BaseProvider`)
- Remove `agents` property as well.